puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/duffy.groovy

```diff
@@ -1,21 +1,39 @@
 def provision() {
     dir('foreman-infra') {
-        git url: 'https://github.com/ehelms/foreman-infra.git', branch: 'ci-centos-org'
+        git url: 'https://github.com/theforeman/foreman-infra.git'
     }
 
-	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
+  	dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
         runPlaybook('provision.yml', 'localhost')
     }
 }
 
 def deprovision() {
-    if(fileExists('foreman-infra')) {
+    if (fileExists('foreman-infra')) {
         dir('foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible') {
             runPlaybook('deprovision.yml', 'localhost')
-        }
+      	}
     }
 }
 
 def cico_inventory(relative_dir = '') {
-	return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/cico_inventory'
+    return relative_dir + 'foreman-infra/puppet/modules/jenkins_job_builder/files/centos.org/ansible/cico_inventory'
```

Comment:
In nightlyTest you do use the `ci` symlink and I think it would be useful here too. Would that work? (same for ssh_config below)

Reply:
I think so, yes, @ehelms are you okay with this change? I know you wrote this originally.

Reply:
I think that's fine.